### ISSUE TITLE
Prevent transition on screen reszie

### ISSUE THREAD

**Issue by graforlock** [pawelgrzybek/siema#85]
When you resize the screen slides literally go mental in the attempt to re-position themselves.  It doesnt look  happy UX-wise.

I believe its better to remove the 200ms on resize used to slide transition to prevent that from happening?

**Comment by pawelgrzybek** [comment]
Hi.

It is looking pretty smooth and elegant to me. What in particular you consider as a bad UX in this functionality?

![siema](https://user-images.githubusercontent.com/3697104/30536898-a0e9aac2-9c5f-11e7-8f7c-0ffac6839fdf.gif)

**Comment by graforlock** [comment]
I will fork the pen later on and reproduce issue.

**Comment by pawelgrzybek** [comment]
Thanks…

**Comment by graforlock** [comment]
It doesn't occur on initial slide, its all smooth and elegant, but when you already go to the next page, for example:

https://codepen.io/graforlock/pen/wrMGxb

It then gets wobbly and mental; its especially visible if carousel is big enough and images are bigger. Its is not a massive issue but eliminating transitions on resize prevents it from happening.

**Comment by pawelgrzybek** [comment]
Thanks for your example. I get your point now. It is a fair suggestion but I wouldn't call it an issue or a bug. I'll bare this in mind and check if there is an lightweight solution to implement it.

Thanks again!

**Comment by graforlock** [comment]
Well yeah, technically its an enhancement. For clarity, I didn't say its bad UX-wise, I said its not happy :))

**Comment by jochemkeller** [comment]
Hey guys, just want to chime in and show you my hacky solution on how I got around this problem. What I did is put a HTML class on the document.body WHILE a resize is being done, which will let me override the transition within my CSS. This is the snippet that adds the HTML class to the document.body:

```
var resizeTimer;

window.addEventListener('resize', function() {

	document.body.classList.add('is-resizing');

	clearTimeout(resizeTimer);

	resizeTimer = setTimeout(function() {
		document.body.classList.remove('is-resizing');
	}, 100);

});
```

Then, in the (S)CSS for my Siema slider I put the following snippet:

```
.slider {

        // This is the div that's used as the "slider-track", and therefor has the transition properties on it.
	> div {

		.is-resizing & {
			transition: none !important;
		}

	}

}
```

Et voila, snappy resizing. It's a bit hackish and could be improved a lot I imagine, but maybe this puts @pawelgrzybek on the right track as to building this into Siema itself? Remove the transition properties while a resize is happening, and add them back once the resizing is done. I don't have the time to pull request now unfortunately, but I'm sure it could be added easily! :-)
